### PR TITLE
BL-1170, switch back to using Acrobat

### PR DESCRIPTION
**Pull request by JohnThomson**
Display and print PDFs using Acrobat if installed.
This avoids various bugs in PDFjs (also BL-1037, 1170, maybe others).
Unfortunately it's not an option on Linux.

### CODE REVIEW

File: src/BloomExe/Publish/PdfViewer.cs [hunk 1]
```diff
@@ -24,17 +24,26 @@ public PdfViewer()
...
-			{
-				_pdfViewerControl = new AdobeReaderControl();
-			}
-			else
+			// In Windows we would prefer to use Acrobat to display and print PDFs. It avoids various bugs in
+			// PDFjs, such as BL-1177 (Andika sometimes lost when printing directly from Bloom),
+			// BL-1170 Printing stops after certain point
+			// BL-1037 PDFjs sometimes fails to display if use certain jpg images
+			// If Acrobat is not installed, it will fall back to PDFjs, and we hope for the best.
+			// Todo: we need a better solution in Linux, also. Ghostscript might provide something but
```
Comment (StephenMcConnel) [inline src/BloomExe/Publish/PdfViewer.cs]: Does AdobeReaderControl() itself fall back to using GeckoWebBrowser() internally?

File: src/BloomExe/Publish/PdfViewer.cs [hunk 2]
```diff
@@ -85,7 +112,8 @@ public bool ShowPdf(string pdfFile)
 		public void Print()
 		{
 #if !__MonoCS__
-			if (Settings.Default.UseAdobePdfViewer)
+			var arc = _pdfViewerControl as AdobeReaderControl;
+			if (arc != null)
 			{
 				((AdobeReaderControl)_pdfViewerControl).Print();
```
Comment (StephenMcConnel) [inline src/BloomExe/Publish/PdfViewer.cs]: Does viewing always precede printing, so that if we get here we know that Adobe Reader is installed?

**Comment by StephenMcConnel** [comment]
If the answer to my one question is "yes", then LGTM.  Otherwise, another fallback operation may be needed if feasible.

**Comment by JohnThomson** [comment]
I added a comment about your first issue. But now I've added another commit to this PR.

**Comment by StephenMcConnel** [comment]
:+1:

**Pull request merged**